docker-for-aws/index.md

```diff
@@ -11,9 +11,11 @@ redirect_from:
 {% include d4a_buttons.md %}
 
 ## Docker Enterprise Edition (EE) for AWS
-This deployment is fully baked and tested, and comes with the latest Enterprise Edition version of Docker. <br/>This release is maintained and receives <strong>security and critical bugfixes for one year</strong>.
+This deployment is fully baked and tested, and comes with the latest Enterprise
+Edition version of Docker. <br/>This release is maintained and receives
+<strong>security and critical bugfixes for one year</strong>.
```

Comment:
Can we use markdown instead of html here?

Reply:
@joaofnfernandes yes, but really the whole Docker for AWS docset needs to be updated for that, I can do it as a part of this PR